app/lib/features/main/app_shell.dart

```diff
@@ -38,6 +39,28 @@ final _log = Logger('a3::config::home_shell');
 
 final ScreenshotController screenshotController = ScreenshotController();
 
+class _TopLevelShell extends ConsumerWidget {
```

Comment:
put that loading into separate widget that is also "watching", so we keep it in memory.